docs_user/modules/openstack-tls_everywhere.adoc

```diff
@@ -0,0 +1,203 @@
+kind: Issuer
+metadata:
+  name: selfsigned-issuer
+  namespace: openstack
+spec:
+  selfSigned: {}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: rootca-internal
+  namespace: openstack
+spec:
+  isCA: true
+  commonName: rootca-internal
+  secretName: rootca-internal
+  privateKey:
+    algorithm: RSA
+    size: $KEY_LENGTH
+  issuerRef:
+    name: selfsigned-issuer
+EOF
+----
+
+
+OK, you just created the CRs with the initial content. Now you will need to import the certificate and the key that you exported from FreeIPA. You can do this with the `oc patch` command. For the CA certificate (correspondeing to the `ca.crt` item on the Certificate, type:
+
+----
+oc patch secret rootca-internal -n openstack -p="{\"data\":{\"ca.crt\": \"`$IPA_SSH openssl pkcs12 -in /tmp/freeipa.p12 -passin file:/etc/pki/pki-tomcat/alias/pwdfile.txt -nokeys | openssl x509 | base64 -w 0`\"}}"
+----
+
+You will also need to update the `tls.crt` item. For the CA, this will receive the very same contents of `ca.crt`. The command just needs a small adjustment:
+
+----
+oc patch secret rootca-internal -n openstack -p="{\"data\":{\"tls.crt\": \"`$IPA_SSH openssl pkcs12 -in /tmp/freeipa.p12 -passin file:/etc/pki/pki-tomcat/alias/pwdfile.txt -nokeys | openssl x509 | base64 -w 0`\"}}"
+----
+
+Finally, update the key:
+
+----
+oc patch secret rootca-internal -n openstack -p="{\"data\":{\"tls.key\": \"`$IPA_SSH openssl pkcs12 -in /tmp/freeipa.p12 -passin file:/etc/pki/pki-tomcat/alias/pwdfile.txt -nocerts -noenc | openssl rsa | base64 -w 0`\"}}"
+----
+
+Now create the cert-manager Issuer, referencing the created secret:
+
+[source, yaml]
+----
+oc apply -f - <<EOF
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openstack
+---
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: rootca-internal
+  namespace: openstack
+  labels:
+    osp-rootca-issuer-internal: ""
+spec:
+  ca:
+    secretName: rootca-internal
+EOF
+----
+
+==== Checking the newly provisioned CRs
+
+You can check the created resources with the commands below:
+
+----
+oc get issuers -n openstack
+----
+
+----
+oc get secret rootca-internal -n openstack -o yaml
+----
+
+== Stop and disable certmonger, and remove the existing certificates
+
+The final step on this activity is to stop and disable the certmonger service, and remove all certificates managed by it. The code below accomplishes this task:
+
+[source, bash]
+----
+#!/bin/bash
+
+if sudo systemctl is-active certmonger.service; then
+    echo "Stopping certmonger..."
+    sudo systemctl stop certmonger.service
+    echo "Disabling certmonger..."
+    sudo systemctl disable --now certmonger.service
+fi
+
+# If necessary, adjust the directory below accordingly.
+certs_directory=/var/lib/certmonger/requests/
+certs_to_stop=$(ls -1 $certs_directory)
+number_of_certs=$(ls -1 $certs_directory | wc -l)
+
+if [ $? != 0 ] || [ $number_of_certs = 0 ]; then
+    echo "No certificates to be removed."
```

Comment:
It's not removing the certificates, but tracking requests for the certificates

Reply:
As ditto above, stopping to track a certificate has the same effect as removing it, according to the information provided by the FreeIPA team.